contract-tests/conftest.py

```diff
@@ -0,0 +1,44 @@
+# Configuration file for running contract-tests
+import pytest
+import requests
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--server",
+        dest="server",
+        default="http://localhost:8000",
+        help="Server to run tests against"
+    )
+    parser.addoption(
+        "--no-verify",
+        action="store_false",
+        dest="verify",
+        default=None,
+        help="Server to run tests against"
+    )
+
+
+@pytest.fixture
+def conf(request):
+    return request.config
+
+
+class RequestVerifyWrapper(object):
+    """
+    `request.Session` doesn't have a way to permanently change the verify flag. This provides that.
```

Comment:
Untrue, they have a `verify` attribute that can be set to false:

```python
session = requests.Session()
session.verify = False
session.get('https://localhost:8000')
```

I tested this in my normandy virtualenv and it seemed to work.